src/VisualStudio/Core/Next/Diagnostics/OutOfProcDiagnosticAnalyzerExecutor.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
hmmm this is also move. not sure why it shows up as new file. might be due to I pulled upstream master which has change on this file?